auth-api/src/auth_api/utils/user_context.py

```diff
@@ -119,14 +105,27 @@ def token_info(self) -> Dict:
         """Return the name."""
         return self._token_info
 
+    @property
+    def account_id(self) -> Dict:
+        """Return the account id."""
+        account_id = _get_token_info().get('Account-Id', None)
```

Comment:
the only time we have account id is the authorization request...
hope nobody will use this method in other cases where we need an org id

Reply:
Yea, I think in most of the other cases we have org_id in the payload or in path.